src/actions/childrenRequestedThunk.tsx

```diff
@@ -47,7 +47,7 @@ export const childrenRequestedThunk = (id: number) => {
 
         dispatch(childrenRequested());
 
-        const url: string = 'http://localhost:5000/node/' + id.toString() + '/children';
+        const url: string = 'http://localhost:5000/entities/' + id.toString() + '/children';
```

Comment:
This should be a variable later, but I guess we can use this as a starting point.